index.js

```diff
@@ -1,17 +1,77 @@
-const fs = require('fs');
-const path = require('path');
-
 module.exports = {
-  bundle: {
-    directory: 'modules',
-    modules: getBundleModuleNames()
+  improve: '@apostrophecms/login',
+  init(self) {
+    self.apos.template.append('head', '@apostrophecms/login:recaptcha');
```

Comment:
This is a high burden to add to every page. Can this not be loaded dynamically by the Vue component when actually in use?

Reply:
It seems like Vue really doesn't want you to include external script links in components. [There are workarounds](https://medium.com/@gaute.meek/how-to-add-a-script-tag-in-a-vue-component-34f57b2fe9bd) (and the createElement approach), though they feel a bit hacky. Do you think that'd be better?